docs/linux-workstations/index.md

```diff
@@ -0,0 +1,166 @@
+# Introduction 
+
+This page attempts to outline how you can set up your linux workstation VM to run under VMWare Player Pro V15.0.2 or greater.
+
+
+## Obtaining VMWare player 
+
+If your laptop doesn't have VMware player installed, in the first instance open Software center from the start menu and see if it is available. If it is, install it from here. If it is not available for you in Software Center, speak to service desk (or ideally Matt Page, as he managed this rollout originally) to have this added to your software center.
+
+## Obtaining the VM image 
+
+<u>The first step of the process is to access the image which is held on:</u>
+
+<code>\\\\business.ukho.gov.uk\dfs\Prod_Prod\Mint\OVF Image</code>
+
+<code>U:\Prod_Prod\Mint\OVF Image</code>
+
+and copy this directory to your laptop.
+
+<u>Once copied, start up VMware Player Pro and on the "Home Screen" select the option "Open a Virtual Machine":</u>
+
+![VMWare workstation](Linux_workstations_importvm.png)
+
+Please use the file browser displayed to select the location where the image was copied.
+
+You will recieve a dialog referring to a mismatch of versions, this is a known bug in VMWare workstation, press Retry and the process will continue.
+
+<u>You will be asked for a name and a storage path - please ensure that the name of the VM image is:</u>
+
+* UKHO CentOS 7
+
+<u>Please ensure that the storage path of the VM image is:</u>
+
+* C:\VirtualMachines\DevVM
+
+
+'''<u>NOTE:</u> It is important that the specified location is used for compliance with SyOps!'''
```

Comment:
Use a notice to show this https://squidfunk.github.io/mkdocs-material/extensions/admonition/#danger